substrate/frame/contracts/fixtures/drain.wat

```diff
@@ -33,13 +39,32 @@
 			)
 		)
 
-		;; Try to self-destruct by sending full balance to the 0 address.
+		;; Get the minimum balance.
+		(call $seal_minimum_balance (i32.const 16) (i32.const 24))
+
+		;; Minimum balance should be encoded as a u64.
+		(call $assert
+			(i32.eq
+				(i32.load (i32.const 24))
+				(i32.const 8)
+			)
+		)
+
+		;; Make the transferred value exceed the balance by adding the minimum balance.
+		(i64.store (i32.const 0)
+			(i64.add
+				(i64.load (i32.const 0))
+				(i64.load (i32.const 16))
+			)
+		)
```

Comment:
isn't it simpler not to modify the fixture but to add ED to the transferred balance?

Reply:
I mean the fixture is broken, given `seal_balance` should only return free balance: It tries to send away all free balance but expects the transfer to fail. Hence we need to modify it anyways.